src/ui/server.ts

```diff
@@ -20,12 +20,14 @@ import * as i18n from 'lib/service/translation'
 import {ProfileChecker} from 'lib/ui/profileChecker'
 import * as template from 'lib/ui/template'
 
+import * as maintenanceMiddleware from 'lib/service/maintenance'
```

Comment:
as per naming convention, I think people are more familiar with `xxxxService` :-) can we call it `maintenanceService` ? be in line with its parent folder name i.e. `lib/service`.

Reply:
sure